src/routes/data-decode/pipes/get-data-decoded.dto.validation.pipe.ts

```diff
@@ -27,7 +32,9 @@ export class GetDataDecodedDtoValidationPipe
     try {
       return this.genericValidator.validate(this.isValid, data);
     } catch (err) {
-      err.status = HttpStatus.BAD_REQUEST;
+      if (err instanceof HttpException) {
+        Object.assign(err, { status: HttpStatus.BAD_REQUEST });
+      }
```

Comment:
This is a temporary workaround as the way we are setting the `status` is incorrect (it is a readonly property). We should revisit the generic validator to throw `HttpStatus.BAD_REQUEST` directly.